mods/[Tsunamods] Menu +/mod.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<Mod>
+  <ID>44a6896d-96ce-493f-bbb3-35bd29dcf205</ID>
+    <Name>[Tsunamods] Menu +</Name>
+    <Author>Digixu + JamesTheCat</Author>
+  <Category>User Interface</Category>
+<Description>
+4x Menu upscale with background colour & Gradient options!
```

Comment:
The `&` character will create issues ( not sure why the bot job didn't caught it, but I manually lint the xml file it returns an error on this char ). Can you please replace it with `and`?